print-ptp.c

```diff
@@ -203,6 +204,24 @@ static const struct tok ptp_msg_type[] = {
     { 0, NULL}
 };
 
+/* Values from IEEE1588-2008: 13.3.2.10 controlField (UInteger8) */
+#define C_SYNC              0x0
+#define C_DELAY_REQ         0x1
+#define C_FOLLOW_UP         0x2
+#define C_DELAY_RESP        0x3
+#define C_MANAGEMENT        0x4
+#define C_OTHER             0x5
+
+static const struct tok ptp_control_field[] = {
+    { C_SYNC ,"Sync"},
+    { C_DELAY_REQ ,"Delay_Req"},
```

Comment:
Style: Replace the `' ,'` with `', '`.

Reply:
Oops, sorry for missing this - thank you!

Reply:
Looks like this was done intentionally, to match the style higher in the file defining `ptp_msg_type[]`.  Would you like me to change these too?